src/components/dropdown-menu/__tests__/DropdownMenu.test.js

```diff
@@ -450,129 +450,125 @@ describe('components/dropdown-menu/DropdownMenu', () => {
         });
     });
 
-    describe('componentDidUpdate()', () => {
-        test('should add click and contextmenu listeners when opening menu', () => {
-            const wrapper = mount(
-                <DropdownMenu>
-                    <FakeButton />
-                    <FakeMenu />
-                </DropdownMenu>,
-            );
-
-            const instance = wrapper.instance();
+    describe.each([[false], [true]])(
```

Comment:
This is difficult to read. Let's leave the `describe` blocks as regular text and use `test.each`, instead. Or we can just write separate tests for the different prop values rather than iterating.

Reply:
will fix

Reply:
Hi Jared, I looked over the potential fix for this. If I keep existing describe block and use test.each, the diff will look a lot harder. But I made that change as we have 173 test.each and 4 describe.each so test.each is more prevalent. Just be aware the diff will be harder

Reply:
@azuruce, rather than iterating over every test with different permutation of props, can we just create a few new tests for these new props? I don't think the former strategy is scalable as props are added.